### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by DhuruvSingh**
### Desktop Version

- **Implemented Desktop Version using Media Query**
1. Used Grid
2. Used Flexbox
3. Resource Images 

> Please review and give your valuable feedback

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,21 +9,33 @@
...
+
     <section>
         <div class="headline-section">
             <div class="illustration-mobile">
-                <img src="./icons/header-illsutration-mobile@2x.png" alt="">
+                <!-- Mobile Version Illustraion -->
+                <img class="mobile-version-illustration" src="./icons/header-illsutration-mobile@2x.png" alt="">
+                <!-- Desktop Version Illustraion -->
+                <img class="desktop-version-illustration" src="./desktop-version-files/Header-llustration-desktop@2x.png" alt="desktop-version-illustraion">
             </div>
```
Comment (1GORDON) [inline index.html:39]: Great job @DhuruvSingh 🤝

- [ ] [Optional] You may please try to maintain the Figma design as much as possible. You may reduce the space between the paragraph and the social icons. Also, try to remove the green line just below the `h2` My recent works.
- Your design
Your design

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @DhuruvSingh ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Professional Pull Request ✔️
- Linters are passing ✔️
- Responsive design ✔️
- Professional Readme ✔️

## Optional suggestions

- [ ] [Optional] It is advisable to always include node_modules in your `.gitignore` file

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DhuruvSingh) [reply index.html:39]: Sure I will do that sir

**Comment by DhuruvSingh** [comment]
Thankyou So Much For your valuable feedback @1GORDON

**Pull request merged**